backend/src/main/java/com/redhat/cloud/notifications/routers/handlers/userconfig/UserConfigResource.java

```diff
@@ -103,6 +112,11 @@ public Response saveSettingsByEventType(@Context SecurityContext sec, @NotNull @
                         if (eventType.isPresent() && !eventType.get().isSubscriptionLocked()) {
                             // for each email subscription
                             eventTypeValue.emailSubscriptionTypes.forEach((subscriptionType, subscribed) -> {
+                                boolean supported = isTemplateSupported(bundleName, applicationName, eventType.get(), subscriptionType);
+
+                                if (!supported) {
+                                    throw new NotFoundException(String.format("Event type '%s' doesn't support '%s' subscription", eventType.get().getId(), subscriptionType.name()));
```

Comment:
This will likely never be thrown because the frontend will not make it look possible, right?